File: Appendices/Appendix K Source Code/Tier1Administrator(Java)/src/shared/Request.java
package shared;

public class Request {
    private String requestType;         //GET, CREATE, UPDATE, DELETE
    private String requestInformation;  //Employees, Customers, Invoices, ect.
    private String requestContent;      //all, specific id, specific area

    public Request(String type, String info, String content)
    {
        requestType = type;
        requestInformation = info;
        requestContent = content;
    }
}

File: Appendices/Appendix K Source Code/Tier1TruckDriver(Blazor)/TruckDriveTier1/Data/Request.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TruckDriveTier1.Data
{
    public class Request
    {
        public string requestType { get; set; }         //GET, CREATE, UPDATE, DELETE
        public string requestInformation { get; set; }  //Employees, Customers, Invoices, ect.
        public string requestContent { get; set; }      //all, specific id, specific area
    }
}
